### ISSUE TITLE
Consider using new framework

### ISSUE THREAD

**Issue by GalvinGao** [hacdias/caddy-filemanager#55]
Materializecss is the material framework that makes you crazy because it is really handy.
They have great components, grids and so on.
Their project page can be found [here](http://materializecss.com/).

Have a nice day!

**Comment by hacdias** [comment]
Hello @GalvinGao,
Thanks for the suggestion. I like the appearance of that framework, but I ask you, what are the advantages? Right now, I'm not using any framework to keep everything the lightest I can. What do you think?

**Comment by GalvinGao** [comment]
That's a point! I agree with you about the light. So may be we can pick a part of the framework or just reference it so we may have more feature about it. What do you think of that?~
U r doin a really good job. It helps me to build my home file server without building a massive php on my iMac. I'm really appreciate it! Thanks!

**Comment by hacdias** [comment]
Hello @GalvinGao!  I'm not thinking about using materialize.css. Though, a lot of improvements were made in the last days. Not every new feature is working yet, but I'd like you to test if you could. 😄 

*Hint: press F1.*

**Comment by GalvinGao** [comment]
Wow I will try it! But, should I just download a new version on caddyserver.com or I should kind of build it by myself?

**Comment by GalvinGao** [comment]
Oh I found it at [here~](https://github.com/hacdias/caddy-filemanager/commit/ae54ba4ce48c1e4bb374894acc11c07bc3edfd65#diff-62bbab3e3cd7b0f02c5722fe62089badR172)

**Comment by GalvinGao** [comment]
Actually that's a good update! I loves shortcuts, always 😁

**Comment by hacdias** [comment]
Hey Galvin!

To try the new version you should build it from source. If can't or don't
have availability to do it, just tell me your OS and the architecture and
Ill build it for you and send you the binary so you can test it :)

Happy New Year!

**Comment by GalvinGao** [comment]
I'm in OS X El Capitan 10.11.6 (Unix-Like) x64. If you have a binary that's perfect! Thanks!

**Comment by hacdias** [comment]
Thanks @GalvinGao! Here's the link: https://www.dropbox.com/s/tptwcsnp995vrlb/caddy_filemanager_darwin_amd64?dl=0

**Comment by GalvinGao** [comment]
Thanks! I'm doing the test now, wait a minute and I will send you some screenshots!

**Comment by hacdias** [comment]
Sure!

**Comment by hacdias** [comment]
You *may* find some similarities with Google Drive interface 😆

**Comment by GalvinGao** [comment]
Ummm I think I have some problem with the file location. I will send you a screenshot

**Comment by GalvinGao** [comment]
![snip20170101_6](https://cloud.githubusercontent.com/assets/12567059/21580848/fe096f7c-d04e-11e6-9620-6dafc0eecc82.png)

**Comment by hacdias** [comment]
Sorry :( It was my fault!!!

**Comment by hacdias** [comment]
Wait a bit please

**Comment by GalvinGao** [comment]
That's ok. This is the mistake that everyone will make 😝

**Comment by GalvinGao** [comment]
My Caddyfile

`192.168.1.20:2017
filemanager /file {
    show           /Volumes/
    allow_new      true
    allow_edit     true
    allow_commands true
}`

**Comment by hacdias** [comment]
Not your fault! It was my problem :P I'm going to reupload the binary.

**Comment by hacdias** [comment]
I forgot `go generate`. I was using the `debug` option and it was linking to my file system.

**Comment by GalvinGao** [comment]
😝

**Comment by hacdias** [comment]
Here @GalvinGao : https://www.dropbox.com/s/ksqpeejkuk5r7b4/caddy?dl=0

**Comment by GalvinGao** [comment]
Thanks! I'm downloading it~

**Comment by hacdias** [comment]
Cool! Then tell me what you think! There are some features that aren't available yet. And some stuff doesn't work on mobile (yet!).

**Comment by hacdias** [comment]
But I think it is better than the last version 😄

**Comment by GalvinGao** [comment]
At all you have saved me from configuring the nginx and php's filelist. You are just, like an angel.
I'll have a look! Thanks so much for sending me this!

**Comment by hacdias** [comment]
Hahahaha. Thanks and you're welcome 😄 I really like this project and want to make it even better.

**Comment by GalvinGao** [comment]
![snip20170101_7](https://cloud.githubusercontent.com/assets/12567059/21580878/650cbc0e-d051-11e6-84e6-f3a73c093ddb.png)

I'm fall in love.
Reallllllllllllly beauuuuuuuuuutiful!!!!!!!!!!!

**Comment by hacdias** [comment]
Thanks! Though I took my inspiration from Google Drive. We should not forget it 😆 Try changing the view type! 

The sorting in "mosaic" type is not available yet, but it is already possible in detailed mode.

**Comment by GalvinGao** [comment]
![img_1631](https://cloud.githubusercontent.com/assets/12567059/21580904/fec12c3a-d052-11e6-8a99-29fd835eadc4.PNG)

It just cost 5s loading in bad 3G (~100kb/s)!
Good performance!

**Comment by hacdias** [comment]
Cool! I haven't tested it in those terms but I think it can be improved using minified versions of CSS and JS for example :p

**Comment by GalvinGao** [comment]
OK I finally got a problem:

As long as I'm using my iPad more than using computer, the file seems only selectable with my iPad and not being redirect to the file UI. Same thing occurs on folders. I'm doing js track on my computer to try to get help for you with this problem.

Please check it out a bit.

**I might going out with my parents, so it might have a little lonely time with yourself. I will be back in maybe 2~3 hours. Have a nice day!**

**Comment by hacdias** [comment]
@GalvinGao yup. I have an iPhone and a Nexus 7. On Nexus (Chrome) it works just like in desktop and I was really glad. On iPhone it didn't. I'm not sure if it's because of Safari but I already know there's that problem. I'll open an issue and put everything mobile-related in there. 😄 Thanks!

Have a nice day! I'll leave too. Gonna visit my grandma and going to take more than 3 hrs :P

**Comment by hacdias** [comment]
Though I'm closing this issue because this is already 'solved'. Let's move to another issue. Anything you find that could be improved on mobile, put on #59 please.
